src/main/java/com/solr/clientwrapper/domain/port/api/kafkaSender.java

```diff
@@ -0,0 +1,7 @@
+package com.solr.clientwrapper.domain.port.api;
+
+public interface kafkaSender {
+	
+	public String AddToQueue(String payload);
```

Comment:
camelCase naming convention should be followed for method names

Reply:
Camel case - "addToQueue"

Reply:
done